Add clarification on supported NGINX Ingress distros (#10705)
DataDog/integrations-core: nginx_ingress_controller/README.md

## Overview

> This check monitors the kubernetes [NGINX Ingress Controller][1]. To monitor the F5 NGINX Ingress Controller, set up the [Datadog Prometheus integration][10] to monitor metrics of interest available from the list at [NGINX Prometheus Exporter][11]

Reviewer: ```suggestion
This check monitors the Kubernetes [NGINX Ingress Controller][1]. To monitor the F5 NGINX Ingress Controller, set up the [Datadog Prometheus integration][10] to monitor desired metrics from the list provided by the [NGINX Prometheus Exporter][11].
```

> https://github.com/nginxinc/nginx-prometheus-exporter#exported-metrics

Reviewer: ```suggestion
```
